mlflow/ml-package-versions.yml

```diff
@@ -100,6 +102,8 @@ keras:
   autologging:
     minimum: "2.3.0"
     maximum: "2.6.0"
```

Comment:
Should we update this to 2.6.2 since 2.6.0 and 2.6.1 seem bugged?

Reply:
Confusingly, `maximum` specifies the maximum supported version for the latest release of MLflow. Our CI will still test all the way up through the current maximum version that's been released on PyPI, as well as one micro release within every major / minor release. Because 2.6.2 has been released, that is the version that will be tested for the 2.6.x major / minor combo. Fortunately, we don't need to change anything else here.

Reply:
awesome!